website_docs/getting-started.md

```diff
@@ -3,552 +3,897 @@ title: "Getting Started"
-
-```go
-// Write writes the n-th Fibonacci number back to the user.
-func (a *App) Write(ctx context.Context, n uint) {
-	var span trace.Span
-	ctx, span = otel.Tracer(name).Start(ctx, "Write")
+func handleRollDice(w http.ResponseWriter, r *http.Request) {
+	// Create a child span called dice-roller that tracks only this function call
+	_, span := tracer.Start(r.Context(), "dice-roller")
 	defer span.End()
 
-	f, err := func(ctx context.Context) (uint64, error) {
-		_, span := otel.Tracer(name).Start(ctx, "Fibonacci")
-		defer span.End()
-		f, err := Fibonacci(n)
-		if err != nil {
-			span.RecordError(err)
-			span.SetStatus(codes.Error, err.Error())
-		}
-		return f, err
-	}(ctx)
-    /* … */
+	value := rand.Intn(6) + 1
+	fmt.Fprintf(w, "%d", value)
 }
-```
 
-With this change any error returned from the `Fibonacci` function will mark that span as an error and record an event describing the error.
+// Wrap the handleRollDice so that telemetry data
+// can be automatically generated for it
+func wrapHandler() {
+	handler := http.HandlerFunc(handleRollDice)
+	wrappedHandler := otelhttp.NewHandler(handler, "rolldice")
+	http.Handle("/rolldice", wrappedHandler)
+}
 
-This is a great start, but it is not the only error returned in from the application. If a user makes a request for a non unsigned integer value the application will fail. Update the `Poll` method with a similar fix to capture this error in the telemetry data.
+func main() {
+	ctx := context.Background()
 
-```go
-// Poll asks a user for input and returns the request.
-func (a *App) Poll(ctx context.Context) (uint, error) {
-	_, span := otel.Tracer(name).Start(ctx, "Poll")
-	defer span.End()
+	tp := newTraceProvider(ctx)
+	defer func() { _ = tp.Shutdown(ctx) }()
 
-	a.l.Print("What Fibonacci number would you like to know: ")
+	otel.SetTracerProvider(tp)
 
-	var n uint
-	_, err := fmt.Fscanf(a.r, "%d\n", &n)
-	if err != nil {
-		span.RecordError(err)
-		span.SetStatus(codes.Error, err.Error())
-		return 0, err
-	}
+	// Register context and baggage propagation.
+	// Although not strictly necessary, for this sample,
+	// it is required for distributed tracing.
+	otel.SetTextMapPropagator(
+		propagation.NewCompositeTextMapPropagator(
+			propagation.TraceContext{},
+			propagation.Baggage{},
+		),
+	)
+
+	tracer = tp.Tracer(serviceName)
 
-	// Store n as a string to not overflow an int64.
-	nStr := strconv.FormatUint(uint64(n), 10)
-	span.SetAttributes(attribute.String("request.n", nStr))
+	wrapHandler()
 
-	return n, nil
+	err := http.ListenAndServe(":8080", nil)
+	if err != nil {
+		log.Fatal(err)
+	}
 }
 ```
 
-All that is left is updating imports for the `app.go` file to include the [`go.opentelemetry.io/otel/codes`] package.
+This will emit a log showing the same trace as before, but now from the
```

Comment:
Remind the reader to re-run the `http://localhost:8080/rolldice` endpoint. Also, you mention that the output is in the collector stdout. Should be worth mentioning that nothing will print to stdout for the app.